src/components/DistanceRequest.js

```diff
@@ -175,13 +205,16 @@ function DistanceRequest({transactionID, transaction, mapboxAccessToken}) {
                             zoom: DEFAULT_ZOOM_LEVEL,
                         }}
                         style={styles.mapView}
+                        waypoints={waypointMarkers}
```

Comment:
After adding waypoints, we should have changed the default location from SF to the first value in waypoints to keep animations consist (coming from https://github.com/Expensify/App/issues/29471)

Reply:
ah that's a good point. Thank for letting us know!